### ISSUE TITLE
Output pins for PF1 in 80mA mode confusing

### ISSUE THREAD

**Issue by hmeijdam** [dbuezas/arduino-web-timers#38]
I am looking for the output pin of OC3A in the LGT8F. The 20mA output is on PD6 (Arduino pin 6, TXD pin) and I assumed the PF1 output is on the same pin. However the Web Timer page shows D1 on the SSOP20 package? D1 is not in the pinout chart. Pin 1 on the chip is te RST pin.  

![image](https://github.com/user-attachments/assets/56c2d525-95e3-42de-9619-d67e619758f0)

**Comment by dbuezas** [comment]
OC3A is indeed on on pin 2, the same pin as reset. You'll have to switch the functionality of the pin to use it, but beware it gets trickier to flash after because of that.

[
![Uploading Screenshot_20241208-191424.png…]()

**Comment by dbuezas** [comment]
![Screenshot_20241208-191424](https://github.com/user-attachments/assets/8383eb6f-7cc4-4c71-8af3-f71c8cee3d57)

**Comment by hmeijdam** [comment]
Now I am even more confused.  I see OC3A also in  PD6, I would have expected that, 

![image](https://github.com/user-attachments/assets/1e78aadf-1efa-4158-9231-0e959b314025)



as it is the 80mA pin according to the pin layout picture that [Wolfgang](https://github.com/wollewald) created

![lgt8f328p_pro_mini_ssop20_pinout](https://github.com/user-attachments/assets/1f3101f4-0cc0-42ce-8207-3e36832099de)

**Comment by dbuezas** [comment]
Yes, the timer has two gpios it can output to. PF1 is the high current one on pin 1, the other is PD6.
You'll still have to look in the datasheet how to disable reset and understand that without reset flashing gets trickier (I think you have to manually cut and reconnect power at the right moment)

**Comment by dbuezas** [comment]
Great!
You could also invert the button logic so it doesn't disable reset if the button is pressed, so that you only press it to flash

**Comment by hmeijdam** [comment]
I learned one thing today. When I initialize the timer I **must** set TCCR3A first and only then I can set OCR3A and OCR3B.

I am using TCCR3A to set the beeper on and off in my loop(). 
When I deleted setting TCCR3A in my setup() to leave that to the loop(), I got no sound.

Further experimenting revealed the order of register initialization is relevant. TCCR3A first and then the OCR's . After setting both OCR's I can set TCCR3A to zero and use it as on/off switch.
